tests/imputers/test_MedianImputer.py

```diff
@@ -206,42 +176,6 @@ def expected_df_3():
 
         return df
 
-    def test_check_is_fitted_called(self, mocker):
-        """Test that BaseTransformer check_is_fitted called."""
-        df = d.create_df_1()
-
-        x = MedianImputer(columns="a")
-
-        x.fit(df)
-
-        expected_call_args = {0: {"args": (["impute_values_"],), "kwargs": {}}}
-
-        with ta.functions.assert_function_call(
-            mocker,
-            tubular.base.BaseTransformer,
-            "check_is_fitted",
-            expected_call_args,
-        ):
-            x.transform(df)
-
-    def test_super_transform_called(self, mocker):
-        """Test that BaseTransformer.transform called."""
-        df = d.create_df_1()
-
-        x = MedianImputer(columns="a")
-
-        x.fit(df)
-
-        expected_call_args = {0: {"args": (d.create_df_1(),), "kwargs": {}}}
-
-        with ta.functions.assert_function_call(
-            mocker,
-            tubular.base.BaseTransformer,
-            "transform",
-            expected_call_args,
-        ):
-            x.transform(df)
-
     @pytest.mark.parametrize(
```

Comment:
Thanks for catching this :) I have now removed test_nulls_imputed_correctly_1 and test_nulls_imputed_correctly_2 as well as expected_df_1 and expected_df_2, here: https://github.com/lvgig/tubular/pull/220/commits/2d7b045ca9bf650e83433a60f923af774a2f5748. Regarding testing with and without weights, perhaps it's best to add this functionality as part of https://github.com/lvgig/tubular/issues/221 so that all transformers with weights can have this functionality? So, for now, I have renamed test_nulls_imputed_correctly_3 to test_nulls_imputed_correctly_weights and not deleted it. Let me know if all this makes sense!

Reply:
makes sense, thanks RJ :)